test/generate-py.sh

```diff
@@ -0,0 +1,72 @@
+# Copy this file to compiler test folder where the .npy files are generated.
+# Update SIMULATOR_PATH value and execute it.
+
+SIMULATOR_PATH="/home/marte/Desktop/git_fork/puma-simulator" # simulator root path
```

Comment:
Replace the personal path for something else.
```suggestion
SIMULATOR_PATH="INSERT PATH TO PUMA SIMULATOR HERE" # simulator root path
```